client/lib/happychat/connection-ng.js

```diff
@@ -23,7 +24,7 @@ import {
 	requestTranscript,
 } from 'state/happychat/connection/actions';
 
-const debug = require( 'debug' )( 'calypso:happychat:connection' );
+const debug = debugFactory()( 'calypso:happychat:connection' );
```

Comment:
This should be `debugFactory( 'calypso:happychat:connection' )`. Otherwise you create a debug logger and log a message immediately :)

Reply:
whoops! Nice catch. I'll  fixem